dipy/reconst/mapmri.py

```diff
@@ -108,7 +108,7 @@ def __init__(self,
         >>> data, gtab = dsi_voxels()
         >>> sphere = get_sphere('symmetric724')
         >>> from dipy.sims.voxel import SticksAndBall
-        >>> data, golden_directions = SticksAndBall(gtab, d=0.0015, S0=1, angles=[(0, 0), (90, 0)], fractions=[50, 50], snr=None)
+        >>> data, golden_directions = SticksAndBall(gtab, d=0.0015, S0=100, angles=[(0, 0), (90, 0)], fractions=[50, 50], snr=None)
```

Comment:
While we're at it, would you mind making this PEP8 compliant, by introducing a line-break? 

It should look like this: 

>>> data, golden_directions = SticksAndBall(gtab, d=0.0015, S0=100,
... angles=[(0, 0), (90, 0)], fractions=[50, 50], snr=None)